execution/executor-benchmark/src/native/mod.rs

```diff
@@ -1,4 +1,8 @@
 // Copyright (c) Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+pub mod aptos_vm_uncoordinated;
```

Comment:
Is this the best place to have 'native' module ??
How about having the native module in "aptos-move/aptos-vm/src/block_executor/" ?

Native is used inside of "aptos-move/aptos-vm/src/..." and I suppose it will be used more from 'lower layers'. executor-benchmark seems to be too high level for this. Even in this code, probably need for 'vm_wrapper' comes from native being in executor-benchmark